### ISSUE TITLE
Issue #548 broke sshd config checks without Match blocks

### ISSUE THREAD

**Issue by riaano** [CISOfy/lynis#554]
@mboelen This fix for issue #548 with commit id 84faf57, broke all my SSH checks that does not have a Match blocks

**Before:**

```
[+] SSH Support
------------------------------------
  - Checking running SSH daemon                               [ FOUND ]
    - Searching SSH configuration                             [ FOUND ]
    - SSH option: AllowTcpForwarding                          [ SUGGESTION ]
    - SSH option: ClientAliveCountMax                         [ OK ]
    - SSH option: ClientAliveInterval                         [ OK ]
    - SSH option: Compression                                 [ OK ]
    - SSH option: FingerprintHash                             [ OK ]
    - SSH option: GatewayPorts                                [ OK ]
    - SSH option: IgnoreRhosts                                [ OK ]
    - SSH option: LoginGraceTime                              [ OK ]
    - SSH option: LogLevel                                    [ OK ]
    - SSH option: MaxAuthTries                                [ SUGGESTION ]
    - SSH option: MaxSessions                                 [ SUGGESTION ]
    - SSH option: PermitRootLogin                             [ OK ]
    - SSH option: PermitUserEnvironment                       [ OK ]
    - SSH option: PermitTunnel                                [ OK ]
    - SSH option: Port                                        [ SUGGESTION ]
    - SSH option: PrintLastLog                                [ OK ]
    - SSH option: Protocol                                    [ OK ]
    - SSH option: StrictModes                                 [ OK ]
    - SSH option: TCPKeepAlive                                [ OK ]
    - SSH option: UseDNS                                      [ OK ]
    - SSH option: UsePrivilegeSeparation                      [ OK ]
    - SSH option: VerifyReverseMapping                        [ NOT FOUND ]
    - SSH option: X11Forwarding                               [ OK ]
    - SSH option: AllowAgentForwarding                        [ OK ]
    - SSH option: AllowUsers                                  [ NOT FOUND ]
    - SSH option: AllowGroups                                 [ NOT FOUND ]
```
**After:**

```
[+] SSH Support
------------------------------------
  - Checking running SSH daemon                               [ FOUND ]
    - Searching SSH configuration                             [ FOUND ]
    - SSH option: AllowTcpForwarding                          [ NOT FOUND ]
    - SSH option: ClientAliveCountMax                         [ NOT FOUND ]
    - SSH option: ClientAliveInterval                         [ NOT FOUND ]
    - SSH option: Compression                                 [ NOT FOUND ]
    - SSH option: FingerprintHash                             [ NOT FOUND ]
    - SSH option: GatewayPorts                                [ NOT FOUND ]
    - SSH option: IgnoreRhosts                                [ NOT FOUND ]
    - SSH option: LoginGraceTime                              [ NOT FOUND ]
    - SSH option: LogLevel                                    [ NOT FOUND ]
    - SSH option: MaxAuthTries                                [ NOT FOUND ]
    - SSH option: MaxSessions                                 [ NOT FOUND ]
    - SSH option: PermitRootLogin                             [ NOT FOUND ]
    - SSH option: PermitUserEnvironment                       [ NOT FOUND ]
    - SSH option: PermitTunnel                                [ NOT FOUND ]
    - SSH option: Port                                        [ NOT FOUND ]
    - SSH option: PrintLastLog                                [ NOT FOUND ]
    - SSH option: Protocol                                    [ NOT FOUND ]
    - SSH option: StrictModes                                 [ NOT FOUND ]
    - SSH option: TCPKeepAlive                                [ NOT FOUND ]
    - SSH option: UseDNS                                      [ NOT FOUND ]
    - SSH option: UsePrivilegeSeparation                      [ NOT FOUND ]
    - SSH option: VerifyReverseMapping                        [ NOT FOUND ]
    - SSH option: X11Forwarding                               [ NOT FOUND ]
    - SSH option: AllowAgentForwarding                        [ NOT FOUND ]
    - SSH option: AllowUsers                                  [ NOT FOUND ]
    - SSH option: AllowGroups                                 [ NOT FOUND ]
```

**Comment by aram535** [comment]
Same here ... 

`- SSH option: ClientAliveCountMax                         [ NOT FOUND ]`

```
# grep ClientAliveCountMax /etc/ssh/sshd_config 
ClientAliveCountMax 2
```

**Comment by aram535** [comment]
PR https://github.com/CISOfy/lynis/pull/557 added to fix issue.

**Comment by mboelen** [comment]
Fixed with #557
